users/admin.py

```diff
@@ -38,6 +43,115 @@ class LanguagesSpokenAtHomeInline(admin.TabularInline):
     verbose_name_plural = _("Languages spoken at home")
 
 
+def _generate_children_event_history_markdown(modeladmin, request, queryset):
```

Comment:
```suggestion
def _generate_children_event_history_markdown(
    modeladmin, request, queryset
) -> Tuple[Guardian, str]:
```
With `from typing import Tuple` for added clarity/documentation about return type.